presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -269,20 +269,39 @@ public void testTimestamp()
         // testing this is hard because of https://github.com/prestodb/presto/issues/7122
     }
 
+    @Test
+    public void jsonDataTypeTest()
+    {
+        DataTypeTest dataTypeTest = DataTypeTest.create()
+                .addRoundTrip(jsonDataType(), "{}")
+                .addRoundTrip(jsonDataType(), null)
+                .addRoundTrip(jsonDataType(), "null")
+                .addRoundTrip(jsonDataType(), "123.4")
+                .addRoundTrip(jsonDataType(), "\"abc\"")
+                .addRoundTrip(jsonDataType(), "\"\"")
+                .addRoundTrip(jsonDataType(), "{\"a\":1,\"b\":2}")
+                .addRoundTrip(jsonDataType(), "{\"a\":[1,2,3],\"b\":{\"aa\":11,\"bb\":[{\"a\":1,\"b\":2},{\"a\":0}]}}")
+                .addRoundTrip(jsonDataType(), "[]");
+
+        dataTypeTest.execute(getQueryRunner(), prestoCreateAsSelect("presto_test_json"));
+        dataTypeTest.execute(getQueryRunner(), postgresCreateAsSelect("tpch.postgresql_test_json1"));
```

Comment:
remove `postgresCreateAsSelect`

Reply:
I will write "done" in each comment and you will resolve conversation when you approve it

Reply:
removed postgresCreateAsSelect done